app/lib/backend_api_logic/routing_policy.rb

```diff
@@ -73,7 +73,13 @@ def as_json
         def replace_path
           return {} if config_path.blank?
 
-          { replace_path: "{{uri | remove_first: '#{config_path.path}'}}" }
+          { replace_path: "{{uri | remove_first: '#{liquid_path}'}}" }
+        end
+
+        private
+
+        def liquid_path
+          config_path.path.gsub(/([+-])/, '\1\1')
```

Comment:
maybe add a comment so some time from now we know why we are doing this?

Reply:
Done